### DISCUSSION TITLE
Fullscreen Windowed

### DISCUSSION THREAD

**Discussion by Reutter** [ScratchAddons/ScratchAddons#6101] [Ideas]
I don't know how difficult or even if it's possible for the addon, but it would be really handy to have a button to move the stage to another window, so I can put the stage fullscreen on my monitor, and code fullscreen on my laptop.

**Comment by WorldLanguages** [comment]
Related: #1993

**Reply by WorldLanguages** [reply]
Streaming the stage to a different window might be somewhat easy, but listening to mouse and keyboard events and sending them back to the main editor tab probably gets complicated.
